packages/device_info/lib/device_info.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
```

Comment:
This plugin is independent of the crash reporting use-case, even though it supports the use-case of reporting unhandled Dart exceptions. Firebase Crash Reporting is being rolled into Crashlytics ([announcement](https://firebase.google.com/docs/crash/)). We are more likely to build a [Crashlytics plugin](https://github.com/flutter/flutter/issues/9592) when it becomes feasible. Also, check out our [Sentry.io client](https://pub.dartlang.org/packages/sentry).

Could you please point to an example for how to build a custom crash report?